spec/tic_tac_toe_spec.rb

```diff
@@ -1,14 +1,13 @@
-require '../tic_tac_toe'
+require_relative '../tic_tac_toe'
 require 'pry'
```

Comment:
Since the only instance of `binding.pry` was removed, might as well remove this `require 'pry'` line since `pry` is no longer used in this file.